src/i965_pciids.h

```diff
@@ -185,3 +185,14 @@ CHIPSET(0x5923, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x5927, kbl, kbl,       "Intel(R) Kabylake")
 CHIPSET(0x3184, glk, glk,       "Intel(R) Geminilake")
 CHIPSET(0x3185, glk, glk,       "Intel(R) Geminilake")
+CHIPSET(0x3EA5, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3EA6, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3EA7, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3EA8, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E90, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E91, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E92, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E93, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E94, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E96, cfl, glk,       "Intel(R) Coffeelake")
+CHIPSET(0x3E9B, cfl, glk,       "Intel(R) Coffeelake")
```

Comment:
Please use the right codename 'Coffee Lake' here, could you help to file another patch to fix the name for KBL and GLK?